ods_ci/tasks/Tasks/rhods_olm.robot

```diff
@@ -32,6 +33,7 @@ Can Install RHODS Operator
   ELSE
       Set Global Variable  ${OPERATOR_NAME}  rhods-operator
       Set Global Variable  ${OPERATOR_NAME_LABEL}  rhods-operator
+      Set Global Variable  ${MODEL_REGISTRY_NAMESPACE}    rhoai-model-registries
```

Comment:
do we want to set this already?
we have not get the code in rhoai yet for modelreg at all

Reply:
speculation :P

Reply:
I tested in RHOAI nightlies and its not affecting the DSC deployment (its being deployed properly), so I guess we can add it just to be ready to work once we add the component downstream. WDYT? @zdtsw

Reply:
i am fine if it does not cause any regression failure for rhoai nightly. 
just say the change wont be used for today/this week probably too

Reply:
👍 will wait when the fix is ready and merged, and will run some regressions deeply